compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -164,24 +164,28 @@ function finishIdentifierStmt(Tokenizer tok, string identifier, Position pos, Po
     }
     else if cur == ":" {
         check tok.advance();
-        return finishOptQualIdentifierStmt(tok, identifier, check tok.expectIdentifier(), pos, startPos);
+        Position newNamePos = tok.currentStartPos();
+        string name = check tok.expectIdentifier();
+        return finishOptQualIdentifierStmt(tok, identifier, name, pos, startPos, newNamePos);
     }
-    return finishOptQualIdentifierStmt(tok, (), identifier, pos, startPos);
+    return finishOptQualIdentifierStmt(tok, (), identifier, pos, startPos, startPos);
 }
 
-function finishOptQualIdentifierStmt(Tokenizer tok, string? prefix, string identifier, Position pos, Position startPos) returns Stmt|err:Syntax {
+function finishOptQualIdentifierStmt(Tokenizer tok, string? prefix, string identifier, Position pos, Position startPos, Position namePos) returns Stmt|err:Syntax {
```

Comment:
What is `pos` here? This needs a comment.

Reply:
parameter no longer needed (same value as `startPos`) removed